crates/astria-sequencer/src/ibc/ics20_transfer.rs

```diff
@@ -412,3 +412,21 @@ async fn execute_ics20_transfer<S: StateWriteExt>(
 
     Ok(())
 }
+
+#[cfg(test)]
+mod test {
+    use super::*;
+
+    #[test]
+    fn is_source_test() {
+        let source_port = "source_port".to_string().parse().unwrap();
+        let source_channel = "source_channel".to_string().parse().unwrap();
+        let asset = Denom::from("source_port/source_channel/asset".to_string());
+        // in the case of a transfer in that is not a refund,
+        // we are the source if the packets are prefixed by the sending chain
+        assert!(is_source(&source_port, &source_channel, &asset, false));
+        // in the case of a refund, we are the source if the packets are not
+        // prefixed by the sending chain
+        assert!(!is_source(&source_port, &source_channel, &asset, true));
```

Comment:
These are effectively testing the same thing, right?

Reply:
one is for the refund case and one is for non refund, but yeah they both test that the prefix checking is ok